src/main/webapp/app/course/course-scores/course-scores-excel-row-builder.ts

```diff
@@ -0,0 +1,61 @@
+import { CourseScoresRowBuilder } from 'app/course/course-scores/course-scores-row-builder';
+import { round } from 'app/shared/util/utils';
+
+/**
+ * This interface represents a cell object, which conforms to the Common Spreadsheet Format (CSF).
+ * https://github.com/sheetjs/sheetjs#common-spreadsheet-format
+ */
+export interface CommonSpreadsheetCellObject {
```

Comment:
Maybe you could add the description for the attributes as comments directly here?